src/components/chat/RoomList.js

```diff
@@ -18,34 +18,28 @@ const RoomList = ({
     localStorage.setItem('roomList', JSON.stringify(roomList));
   }
 
-  const styleRoomList = () => {
-    return {
-      display: 'flex',
-      flexDirection: 'column',
-      width: '100%',
-      marginTop: '48px'
-    };
-  };
-
-  const styleRoomName = (passphrase) => {
-    const lightPurple = '#aa6fe3';
-    return {
-      color: passphrase === passphraseFromUrlHash ? lightPurple : 'white',
-      width: '100%'
-    }
-  }
-
   const setPassphrase = (passphrase) => {
     console.log('setPassphrase() -- TODO: Disconnect the WebSocket, change the current URL hash, auth again, connect via WebSocket');
   }
 
+  const changeRoom = (passphrase) => {
+    // just update fragment and reload page for now
+    window.location.assign(window.location.origin + `/#${passphrase}`);
+    window.location.reload(true);
```

Comment:
Does this happen to work on mobile?! Probably not, but... I thought I'd ask.

Reply:
HOLY SHIT, NICE!!! It does!!!

Reply:
I tested in the Android emulator and it worked for me.